### DISCUSSION TITLE
TA 12.0 alpha 2: bad argument #1 to 'char' (value out of range)

### DISCUSSION THREAD

**Discussion by amitbha** [orbitalquark/textadept#385] [Q&A] [answered]
The file's content is `'愿以发心皓月之光明`.
While delete the beginning single quotes using Backspace, the TA output error: `/opt/textadept/modules/textadept/editing.lua:147: bad argument #1 to 'char' (value out of range)`

**Comment by orbitalquark** [comment, marked as answer]
Thanks for the report. This should be fixed with https://github.com/orbitalquark/textadept/commit/a64ef607a21cde1a45e88e8e3c6587b9421a8cc9
